hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/AbstractKeyDeletingService.java

```diff
@@ -229,6 +231,10 @@ private int submitPurgeKeysRequest(List<DeleteBlockGroupResult> results,
     return deletedCount;
   }
 
+  protected OzoneManagerProtocolProtos.OMResponse submitRequest(OMRequest omRequest) throws ServiceException {
+    return OzoneManagerRatisUtils.submitRequest(ozoneManager, omRequest, clientId, callId.incrementAndGet());
```

Comment:
and i think this API is easy to misuse. We need a little more javadoc for the callId so caller is expected to ensure callId is unique for different requests but remains the same for the same request.

Reply:
[HDDS-11807](https://issues.apache.org/jira/browse/HDDS-11807) is another example of misuse.